### DISCUSSION TITLE
Percenatage value in alert

### DISCUSSION THREAD

**Discussion by Shubham1406220** [jertel/elastalert2#1021] [Q&A] [answered]
How can i add the percentage value and query_key value in percentage alert text send to slack?

**Comment by jertel** [comment, marked as answer]
From the [docs](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#percentage-match):

![image](https://user-images.githubusercontent.com/16639611/203312418-93e00565-b0f8-49c4-9d13-d682d185c29e.png)

Specify the query key alert text arg just as you specified it for the `query_key` rule property value.

**Reply by Shubham1406220** [reply]
Thanks working now, 
I am using the percentage match alert and facing a weird issue. Below is my config for percentage rule. Sometimes i get alerted for the percentage match even though there is no match in query. 

name: "Percentage match"
type: percentage_match
index: xxxx*

filter:
- query:
   query_string:
     query: "level:*"
- terms:
   app_name: ["xxxx", "xxx", "xxxx", "xxxx"]

match_bucket_filter:
- query_string:
     query: "level:ERROR"

max_percentage: 70

query_key: app_name

realert:
  minutes: 5

percentage_format_string: "%.2f"
alert_text_args: [ "kibana_discover_url", "app_name" ]
alert_text: "{{ percentage }}"
alert_text_type: alert_text_jinja

percentage_format_string: "%.2f"

**Reply by jertel** [reply]
You'll need to enable debug logging and watch the log output as the alert is triggered. This will give you the response from Elasticsearch, including the query search results and help you better understand why it's triggering the alert.
